packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FLTWebViewFlutterPlugin.h

```diff
@@ -3,6 +3,26 @@
 // found in the LICENSE file.
 
 #import <Flutter/Flutter.h>
+#import <WebKit/WebKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface FLTWebViewFlutterPlugin : NSObject <FlutterPlugin>
```

Comment:
This will make renaming this class (removing the prefix) when eventually converting this plugin to Swift a breaking change. Could we make a stand-alone API function for this instead?